frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts

```diff
@@ -130,8 +130,21 @@ export const gitPage = {
       .get(gitPO.cancel)
       .should('be.enabled')
       .click(),
-  verifyValidatedMessage: () =>
-    cy.get(gitPO.gitSection.validatedMessage).should('have.text', messages.gitUrlValidated),
+  verifyValidatedMessage: (builderImageName = 'Node.js') => {
```

Comment:
the `builderImageName` will have default value of `Node.js`. What if another URL is given? The step should include the `builderImageName` as param. WDYT?

Reply:
Even if we give different url, still the script works fine, because builder image selection doesn't impact on any screen

Reply:
If the validation fails, how the script will know what builder image (nodejs, python, php, etc..) it has to select?

Reply:
1st point - 
We already used param, but the param got default value 'Node.js'. [If we are not passing any parameter. it autoselects the node.js builder image by default]
3rd point -
Based on the scenario, we can identify the test data.